daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/command/Command.scala

```diff
@@ -70,11 +107,35 @@ final case class CreateAndExerciseCommand(
     choiceArgument: Value,
 ) extends ApiCommand
 
+/** Fetch either a template or interface */
```

Comment:
why do we need this? `fetch` is not a ledger API command so afaict we should never need this.

Reply:
Dropped it